src/central-server-audits.rst

```diff
@@ -5,6 +5,26 @@ Server Audit Logs in Central
 
 As of `version 0.6 <https://github.com/getodk/central/releases/tag/v0.6.0-beta.0>`_, ODK Central tracks and logs audit actions for most administrative actions performed on the server. The following actions are logged:
 
+ - **Project Actions**
+
+   - Create
+   - Update Details (name, settings, archival)
+   - Delete
+
+ - **Form Actions**
+
+   - Create
+   - Update Details (state, drafts, publishes, settings)
```

Comment:
"publishes" reads weird to me here. What exactly is it supposed to represent? Published status?

Reply:
like a publish event. not sure how better to phrase it but i agree it's confusing in context.

Reply:
Maybe we do the obvious thing and say publish events? And would the same apply to drafts?